multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/DetermineApplicationServiceBindingActionsStep.java

```diff
@@ -43,19 +44,22 @@ protected StepPhase executeStep(ProcessContext context) throws FileStorageExcept
             context.setVariable(Variables.SHOULD_UNBIND_SERVICE_FROM_APP, false);
             context.setVariable(Variables.SHOULD_BIND_SERVICE_TO_APP, true);
             context.setVariable(Variables.SERVICE_BINDING_PARAMETERS, bindingParameters);
+            getStepLogger().debug(Messages.CALCULATED_BINDING_OPERATIONS_APPLICATION_SERVICE, app.getName(), service, false, true);
             return StepPhase.DONE;
         }
 
         if (shouldRebindService(serviceBindingParametersGetter, existingApp, service, bindingParameters)) {
```

Comment:
I am thinking of one additional debug message befor invocation of shouldRebindService().
It will show start and end time of that method.